src/components/organisms/ProblemSection.jsx

```diff
@@ -32,12 +32,15 @@ function ProblemSection () {
                  The Problem
                 </Typography>
                 <Typography variant="body1" style={{fontWeight: "300", lineHeight: "1.4"}}>
-                  Math, ugh! Many students feel that math is all about tests,
+                  {/* Math, ugh! Many students feel that math is all about tests,
                   grades, homework, and getting correct answers... What if we
                   could flip things around and make math more about learning
                   from mistakes and even having fun? Give
                   {" "}<b><em>RightOn!</em></b>{" "}
-                  a try and find out how!
+                  a try and find out how! */}
+                  Many people think that STEM learning is all about homework, tests, and correct answers.
+                  We flip that idea on its head, enabling students to learn from mistakes while having fun.
+                  Give {""}<b><em>RightOn!</em></b>{""} a try and find out how!
```

Comment:
>{""}

I'm curious what these empty quotes are adding?

Reply:
I don't think there's any important purpose behind the empty quotes. I originally seen it in Daz's original code, so I followed the same order if it mattered. I can remove the quotes as it's unnecessary.